addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -52,7 +52,7 @@ Wysiwyg.include({
         this.options.toolbarHandler = $('#web_editor-top-edit');
 
         // Dropdown menu initialization: handle dropdown openings by hand
-        var $dropdownMenuToggles = this.$('.o_mega_menu_toggle, #top_menu_container .dropdown-toggle');
+        var $dropdownMenuToggles = this.$('.o_mega_menu_toggle, #top_menu_container .dropdown-toggle:not(.o_extra_menu_items_toggle)');
```

Comment:
_Nitpicking/Can ignore:_
Do we really need to create this new class as its parent has a dedicated class?
Can't it be something like
```suggestion
        var $dropdownMenuToggles = this.$('.o_mega_menu_toggle, #top_menu_container .dropdown-toggle').filter(':not(.o_extra_menu_items .dropdown-toggle)');
```
So we don't need the new class?

Reply:
Of course thought about it but went that way, I'll keep it. Seems better to me in the end. Adding a new class only has benefits, it's clearer, more efficient for this search here, more BS-link if you want to add CSS rules on it, there is no extra loading time since it is something the JS adds, etc. But in the end, anything is good of course. It's also better because when converting to non-jquery, it will just be about changing the $ search by `querySelectorAll`. While your solution would need converting to array, filtering, using `matches`, etc.

But FYI:
- You suggest `$('a, b').filter(':not(d e)')`
- => Actually it can be simplified into `$('a, b').not('d e')`
- What I also don't like about this is that you filter over `a` element which you don't need too.